lib/src/credentials.dart

```diff
@@ -207,7 +207,8 @@ class Credentials {
       String secret,
       Iterable<String> newScopes,
       bool basicAuth = true,
-      http.Client httpClient}) async {
+      http.Client httpClient,
+      Map<String, dynamic> additionalBody}) async {
```

Comment:
Documentation?

Reply:
added

Reply:
Does this make any sense? When creating a client from a flow should vendor specific params be passed along in the client?

Reply:
Maybe we need to pass `vendorSpecificParamsForTokenEndPoint` and `vendorSpecificParamsForCodeAuthorizationEndPoint` when creating a `Client` from a flow..

Curious, what is the use-case for this?